NEWS.md

```diff
@@ -33,11 +33,15 @@ Standard library changes
 
 * Added keyword arguments `rtol`, `atol` to `pinv` and `nullspace` ([#29998]).
 
+#### SparseArrays
+
+* `spmatmul` no longer has the obsolete optional keyword argument `sortindices` ([#30372]).
+* performance improvements for `spmatmul` and hence for sparse matrix-matrix multiplication.
```

Comment:
AFAICT we usually don't include things like this in news.

Reply:
Is a line like this ok or should I leve it out completely?
```
* performance improvements for sparse matrix-matrix multiplication ([#30372]).
```

Reply:
@fredrikekre Why do we not mention performance improvements in NEWS? That generally sounds like users would appreciate it.

Reply:
I'm just saying that we have not done it before.

Reply:
Performance improvements can make the difference between usable and unusable in certain application scenarios. What do you recommend?

Reply:
@fredrikekre Fair enough. I think it is a good idea and I will merge it now. I also think that performance improvements are generally good to include across the board. If it is something we decide that we shouldn't do, we can always remove it.